src/posix.mak

```diff
@@ -139,7 +139,9 @@ RUN_BUILD = $(GENERATED)/build HOST_DMD="$(HOST_DMD)" CXX="$(HOST_CXX)" OS=$(OS)
 all: dmd
 .PHONY: all
 
-dmd: $G/dmd $G/dmd.conf
+dmd: $(GENERATED)/build
```

Comment:
if this is required to make this PR work then you're only working around the issue.  This should work as is.

Reply:
I know, just experimenting whether this makes a difference.

P.S.: All commits not titled `build.d: ...` are just temporary changes, I'll discard them once this PR is ready.